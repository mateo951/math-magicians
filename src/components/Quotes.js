import React from 'react';
import style from './style/quotes.module.css';

const Quotes = () => (
  // eslint-disable-next-line max-len
  <h1 className={style.quote}>A language that does not affect the way you think about programming is not worth knowing.</h1>
);

export default Quotes;
