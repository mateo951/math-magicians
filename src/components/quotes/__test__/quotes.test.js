import React from 'react';
import ReactDOM from 'react-dom';
import { render, screen } from '@testing-library/react';
import Quotes from '../Quotes';

let div = null;
beforeEach(() => {
  div = document.createElement('div');
  document.body.appendChild(div);
  render(<Quotes />);
});

describe('Test rendering', () => {
  it('renders with out crashing', () => {
    ReactDOM.render(<Quotes />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders text correctly', () => {
    const calculator = screen.getByText('A language that does not affect the way you think about programming is not worth knowing.');
    expect(calculator).toMatchSnapshot();
  });
});
