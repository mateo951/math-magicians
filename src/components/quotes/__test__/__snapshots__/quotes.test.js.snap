// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test rendering renders text correctly 1`] = `
<h1
  class="quote"
>
  A language that does not affect the way you think about programming is not worth knowing.
</h1>
`;
